github_releaser/cmd/push.py

```diff
@@ -0,0 +1,25 @@
+import click
+import os
+
+from github_releaser import GithubReleaser
+
+
+@click.command(name="push-files", help="Push new files to a branch")
+@click.option("--account", "--a", required=True, help="Account")
+@click.option("--repository", "--r", required=True, help="Repository")
+@click.option("--token", help="GitHub's API token")
+@click.option("--branch-name", "--b", required=True, help="The branch name. Default: master")
+@click.option("--dest-dir", "--d", required=False, help="The destination subfolder inside the repository. Omit if pushing to the root")
+@click.option("--message", "--msg", required=False, help="The message")
```

Comment:
Perhaps here we can add a default like "updated {filename}" where the filename is the name of the files specified in `--files`

Reply:
oh good idea, agree

Reply:
it's just "added {filename}", I haven't implemented yet the functionality to check whether a file exists and update it if exists and creating it if not. Not it just creates a new one. But probably it should update as well to have a solid behaviour. What do you think?

Reply:
I'm not sure how the GitHub API works but I believe that if the file is already present it will update. We can test that out this afternoon.